bin/custom/questions/resources/questions-beans.xml

```diff
@@ -1,20 +1,13 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<!--
- Copyright (c) 2019 SAP SE or an SAP affiliate company. All rights reserved.
--->
-<!--
-	ATTENTION: This is just an example file. You have to edit it according to your needs.
--->
-
 <beans xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="beans.xsd">
-
-	<!--
-		Example Data POJO
-	
-	<bean class="yemptypackage.data.MyPojo">
-		<property name="id" type="String"/>
-		<property name="flag" type="boolean"/>
-	</bean>
-	-->
-
-</beans>
+    <bean class="com.myapp.questions.data.QuestionData">
+        <property name="id" type="java.lang.String"/>
+        <property name="questionCustomer" type="java.lang.String"/>
```

Comment:
as per the task, questionCustomer and answerCustomer should be of type Customer, not plain String